kdc/krb5tgs.c

```diff
@@ -2433,36 +2154,14 @@ tgs_build_reply(astgs_request_t priv,
 	    goto out;
 	}
 
-	/*
-	 * Check that the KDC issued the user's ticket.
-	 */
-	ret = check_KRB5SignedPath(context,
-				   config,
-				   krbtgt,
-				   cp,
-				   &adtkt,
-				   NULL,
-				   &ad_signedpath);
-	if (ret) {
-	    const char *msg = krb5_get_error_message(context, ret);
-	    kdc_log(context, config, 4,
-		    "KRB5SignedPath check from service %s failed "
-		    "for delegation to %s for client %s (%s)"
-		    "from %s failed with %s",
-		    spn, tpn, dpn, cpn, from, msg);
-	    krb5_free_error_message(context, msg);
-            _kdc_audit_addreason((kdc_request_t)priv,
-                                 "KRB5SignedPath check failed");
-	    goto out;
-	}
-
-	if (!ad_signedpath) {
+	if (mspac == NULL || !ad_signedticket) {
 	    ret = KRB5KDC_ERR_BADOPTION;
 	    kdc_log(context, config, 4,
-		    "Ticket not signed with PAC nor SignedPath service %s failed "
+		    "Ticket not signed with PAC, service %s failed "
 		    "for delegation to %s for client %s (%s)"
-		    "from %s",
-		    spn, tpn, dpn, cpn, from);
+		    "from %s,  mspac(%p), ad_signedticket(%s).",
```

Comment:
want to leave this as a pointer? (%p)

Reply:
Yeah, I just want to convey is it's NULL, I'll rework this log.